example/Example.cmake

```diff
@@ -37,9 +37,9 @@
       add_executable(g3log-FATAL-sigsegv ${DIR_EXAMPLE}/main_sigsegv.cpp)
       add_executable(g3log-FATAL-choice ${DIR_EXAMPLE}/main_fatal_choice.cpp)
       
-      target_link_libraries(g3log-FATAL-contract ${G3LOG_LIBRARY} ${EXAMPLE_PLATFORM_LINK_LIBRIES})
-      target_link_libraries(g3log-FATAL-sigsegv ${G3LOG_LIBRARY} ${EXAMPLE_PLATFORM_LINK_LIBRIES})
-      target_link_libraries(g3log-FATAL-choice ${G3LOG_LIBRARY} ${EXAMPLE_PLATFORM_LINK_LIBRIES})
+      target_link_libraries(g3log-FATAL-contract ${G3LOG_SHARED_LIBRARY} ${EXAMPLE_PLATFORM_LINK_LIBRIES})
+      target_link_libraries(g3log-FATAL-sigsegv ${G3LOG_SHARED_LIBRARY} ${EXAMPLE_PLATFORM_LINK_LIBRIES})
```

Comment:
If the default is shared then the rename is not needed. Please remove the change